src/mos_editor.c

```diff
@@ -43,7 +43,7 @@ extern BYTE scrcols;
 
 // Storage for the command history
 //
-static char	cmd_history[cmd_historyDepth][cmd_historyWidth + 1];
+static char	* cmd_history[cmd_historyDepth];

```

Comment:
the approach taken here is to use an array of pointers to strings for history entries

I did consider using a linked list, where we'd just store a pointer to the first history entry.  this would remove the need to shuffle history entries when we've reached our size limit.  for now that approach hasn't been taken